.circleci/config.yml

```diff
@@ -29,11 +29,16 @@ workflows:
           parameters:
             pgversion: ["9.6", "12"]
     - func:
-        name: stage1-func-pg<< matrix.pgversion >>-buster
+        name: stage1-func-pg13-py2-buster
```

Comment:
py2 ? Shouldn't it be "stage1-func-pg13-py3-buster" and "stage1-func-pg11-py2-buster"